ghost/audience-feedback/lib/AudienceFeedbackService.js

```diff
@@ -1,7 +1,21 @@
+const urlUtils = require('../../core/core/shared/url-utils');
+
 class AudienceFeedbackService {
-    buildLink() {
-        // todo
-        return new URL('https://example.com');
+    /**
+     * @param {string} uuid
+     * @param {string} postId
+     * @param {0 | 1} score
+     */
+    buildLink(uuid, postId, score) {
+        const siteUrl = urlUtils.getSiteUrl();
```

Comment:
RE: https://ghost.slack.com/archives/C02G9E68C/p1665739008939039 we should use `urlFor` rather than `getSiteUrl`

Reply:
Fixed :)